### DISCUSSION TITLE
Can we make the output of ripgrep predictable?

### DISCUSSION THREAD

**Discussion by aliasgar55** [BurntSushi/ripgrep#2674] [Q&A] [answered]
The output of ripgrep is not predictable which makes its hard to track if we are reviewing each match and then doing a search again (which I do in vim) to check on the next match. If I can make it more predictable like sort it based on the filename or line number, that would be really helpful
Thank you

**Comment by BurntSushi** [comment, marked as answer]
> If I can make it more predictable like sort it based on the filename

Yes. Use the `--sort path` flag.

It does disable parallelism, so it may be slower. See https://github.com/BurntSushi/ripgrep/issues/152 for more details on that.
